SparkPost/2web2ui
# AC-1265: Multi recipient data privacy tab

### What Changed
 - Added multi recipient tab to data privacy

### How To Test
 - Enable feature flag `data_privacy`
 - Go to /account/data-privacy
 - Go to the tab `Multiple Recipients`
 - Check that the flow works correctly

### To Do
- [x] Fix download link for the error
